docs/src/use/migrate-to-9.0.0.md

```diff
@@ -543,3 +544,11 @@ linter.getRules();
 ```
 
 **Related Issues(s):** [#13481](https://github.com/eslint/eslint/issues/13481)
+
+## <a name="output-file"></a> `--output-file` now writes a file to disk even with empty output
```

Comment:
This section should be after the ``varsIgnorePattern`` section.

Also, I'm not sure but maybe there should be an "an" between "with" and "empty" ("with an empty output")?

Reply:
Updated